students/gyuchan/users/views.py

```diff
@@ -34,4 +36,27 @@ def post(self, request):
             )
             return JsonResponse({'message':'CREATED'}, status=201)
         except KeyError:
-            return JsonResponse({'message':'KEY_ERROR'}, status=400)
\ No newline at end of file
+            return JsonResponse({'message':'KEY_ERROR'}, status=400)
+
+
+class LogInView(View):
+    def post(self, request):
+        # 데이터를 받음
+        data = json.loads(request.body)
+
+        try:
+            
+            user_email    = data['email']
+            user_password = data['password']
+            
+            check_user = User.objects.get(email=user_email)
```

Comment:
```suggestion
            user = User.objects.get(email=user_email)
```
check_user는 함수명에 어울립니다. 하나의 user 인스턴스를 가져오는 코드이니 user가 더 어울리겠네요.